table/block_based/block.h

```diff
@@ -343,6 +343,8 @@ class BlockIter : public InternalIteratorBase<TValue> {
 
   Cache::Handle* cache_handle() { return cache_handle_; }
 
+  virtual void Next() = 0;
```

Comment:
This is also in the parent class, just add it here too to solve some compiler error.